cql-unit-tests/test/treatmentSummary.test.ts

```diff
@@ -52,34 +50,34 @@ describe('treatment summary', () => {
     });
 
     it.each([
-            ['mild', 'use', new ClinicalAssessmentBuilder().withMildSeverity().build()],
-            ['moderate', 'use', new ClinicalAssessmentBuilder().withModerateSeverity().withConcerningLab(1).build()],
+            ['mild', true, new ClinicalAssessmentBuilder().withMildSeverity().build()],
+            ['moderate', true, new ClinicalAssessmentBuilder().withModerateSeverity().withConcerningLab(1).build()],
             ['severe', null, new ClinicalAssessmentBuilder().withSevereSeverity().build()],
             ['critical', null, new ClinicalAssessmentBuilder().withCriticalSeverity().build()],
             ['none', null, {}],
         ]
-    )('For %p severity, Recommend Antibodies Treatment is %p', (severityType: string, expectedRecommendation: string, clinicalAssessmentOverrides: Partial<ClinicalAssessmentsParameters>) => {
+    )('For %p severity, Recommend DischargeHome Treatment is %p', (severityType: string, expectedRecommendation: string, clinicalAssessmentOverrides: Partial<ClinicalAssessmentsParameters>) => {
         const cqlExpressionParameters = buildCQLExpressionParameters(clinicalAssessmentOverrides);
-        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Antibodies Treatment');
+        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Discharge Treatment');
         expect(recommendNonPharma).toEqual(expectedRecommendation);
     });
 
     it.each([
             ['mild', null, new ClinicalAssessmentBuilder().withMildSeverity().build()],
```

Comment:
i defer. and wondering we prefer `false` to `null`. No problem either way to me

Reply:
I agree it's confusing, but currently, treatments are recommended (true), not recommended(false), and not applicable(null). I also deferred the change when we have a better grasp of how `HasAdmissionOrDischargeRecommendation` can be used.